app/controllers/users_controller.rb

```diff
@@ -3,33 +3,41 @@ class UsersController < ApplicationController
   before_action :logged_in_user, only: %i(index edit update destroy)
   before_action :correct_user, only: %i(edit update)
   before_action :admin_user, only: %i(destroy)
+  skip_before_action :verify_authenticity_token
 
 
- def index
+  def index
     @users = User.paginate(page: params[:page])
   end
 
-  def show; end
+  def show
+    @user = User.find_by id: params[:id]
```

Comment:
You should check the exist of user before getting user's microposts

```suggestion
    @user = User.find_by id: params[:id]
    if @user.nil?
      flash[:danger] = "Not found user"
      redirect_to root_url
    end
```

Reply:
Hi.. what do you mean? is it delete that line?

Reply:
I just updated. pls check again. Thank you!